app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useRef } from 'react';
+import { useI18n } from '@18f/identity-react-i18n';
+import { PageHeading } from '@18f/identity-components';
+import BackButton from './back-button';
+
+function InPersonLocationFullAddressEntryPostOfficeSearchStep({ toPreviousStep }) {
+  const { t } = useI18n();
+
+  // ref allows us to avoid a memory leak
+  const mountedRef = useRef(false);
+
+  useEffect(() => {
+    mountedRef.current = true;
+    return () => {
+      mountedRef.current = false;
+    };
+  }, []);
```

Comment:
If you don't expect to use this, then we should remove it.

```suggestion
```

Reply:
Done in ea360c0da